contribution.md

```diff
@@ -0,0 +1,77 @@
+
+### Parameter Containers
+Relevant Headers: [`accept_encoding.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/accept_encoding.h), [`auth.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/auth.h), [`bearer.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/bearer.h), [`body.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/body.h), [`buffer.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/buffer.h), [`callback.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/callback.h), [`cert_info.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/cert_info.h), [`connect_timeout.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/connect_timeout.h), [`cookies.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/cookies.h), [`cprtypes.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/cprtypes.h), [`file.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/file.h), [`http_version.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/http_version.h), [`interface.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/interface.h), [`limit_rate.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/limit_rate.h), [`local_port.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/local_port.h), [`local_port_range.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/local_port_range.h), [`parameters.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/parameters.h), [`low_speed.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/low_speed.h), [`payload.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/payload.h), [`proxies.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/proxies.h), [`proxyauth.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/proxyauth.h), [`range.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/range.h), [`redirect.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/redirect.h), [`reserve_size.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/reserve_size.h), [`resolve.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/resolve.h), [`ssl_options.h`](https://github.com/libcpr/cpr/blob/master/include/cpr/ssl_options.h)
+
+These containers are intended to be provided to API functions in order to generate template instantiations that execute the specific request(s). They contain limited logic, and much of their functionality is contingent on their constructors. They are aimed to be used in order to supply arguments to `CPR` requests in an object-oriented way. To see how they are handled, it usually is a good idea to look at the implementation of the polymorphic `Session::SetOption` method.
```

Comment:
Uh, found an other one: `CPR` 😜

Reply:
cpr being the multifaceted project that it is, I thought we'd better keep the reader on edge by constantly changing the way we refer to it :stuck_out_tongue_closed_eyes: 

That aside, yeah I'm doing some editing, these things will be smoothed out.